discord-scripts/discord-webhook.ts

```diff
@@ -84,7 +80,7 @@ export default async function webhookDiscord(
     if (!userId) throw new Error("User not found with the specified name")
 
     const member = await guild.members.fetch(userId)
-    const currentNickname = member.nickname || member.user.username
+    const currentNickname = member.nickname ?? member.displayName
```

Comment:
Should this then fall back one more time on the username, or does display name handle that? What's the difference between display name and nickname? It feels like that might be backwards (display name being server specific and nick being global maybe?).

Reply:
The nice part about `member.displayName` ([doc link](https://discordjs.dev/docs/packages/discord.js/14.16.3/GuildMember:Class#displayName) is it will load Guild nickname again and then if not default to username.

Reply:
Should we *only* use that in that case?